architectexample/views.py

```diff
@@ -3,4 +3,4 @@
 
 
 def home(request):
-  return HttpResponse("<H1>Welcome to the Polls app!</H1><H2>Navigate to /polls or /admin to get started</H2>")
+  return HttpResponse("<H1>Architect Movie App</H1>")
```

Comment:
Not a big deal but the capitalized `<H1>` that was here before seems weird - I assume it's technically valid html but i've never seen it not lowercase